plan.go

```diff
@@ -134,3 +138,27 @@ func (s *plans) Logs(ctx context.Context, planID string) (io.Reader, error) {
 		logURL: u,
 	}, nil
 }
+
+
+// Provide the JSON version of the plan from the TFE API.
+func (s *plans) JsonOutput(ctx context.Context, planID string) ([]byte, error) {
+	if !validStringID(&planID) {
+		return nil, errors.New("invalid value for plan ID")
+	}
+
+	u := fmt.Sprintf("plans/%s/json-output", url.QueryEscape(planID))
+	// TODO: FInd a way to not have to use our own code to do this - maybe extend the API?
+	req, err := s.client.newRequest("GET", u, nil)
+	if err != nil {
+		return nil, fmt.Errorf("tfe: unable to get json plan %v", err)
+	}
```

Comment:
When handling request errors elsewhere, we generally return the error from the API directly:

```
	if err != nil {
		return nil, err
	}
```

Is there a special reason we need extra error text here?

Reply:
I've updated this to conform to your design preferences.